generate-err-logs.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+
+main() {
+    echo "Generating error logs" >&2
+    generate_logs
+    echo "Generated logs sucessfully" >&2
+}
+
+generate_logs() {
+    pod_definitions_file="failed-pods-definitions.yaml"
+    pod_logs_file="failed-pods-logs.log"
+    event_messages_file="failed-deployment-event-log.log"
+    > $pod_logs_file > $pod_definitions_file > $event_messages_file
```

Comment:
what does it do?

Reply:
reset the files' contents; it is not much useful for the CI as it is for local use.